assets/js/blocks/reviews/reviews-by-category/edit.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+import { ReviewsByCategoryEditorProps } from './types';
```

Comment:
```suggestion
import type { ReviewsByCategoryEditorProps } from './types';
```

Given that `ReviewsByCategoryEditorProps` is a type, using `import type ...` can lead to a slightly faster compile time.